etl/steps/data/garden/wb/2024-12-17/bilateral_remittance.py

```diff
@@ -0,0 +1,75 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+REGIONS = [reg for reg in geo.REGIONS.keys() if reg != "European Union (27)"]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("knomad")
+    ds_regions = paths.load_dataset("regions")
+    ds_income_groups = paths.load_dataset("income_groups")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow.read("knomad")
+
+    # Filter only on bilateral remittances
+    tb = tb[tb["indicator_id"] == "WB.KNOMAD.BRE"]
+
+    # Rename and drop columns
+    tb = tb.drop(columns=["indicator_id", "economy_iso3", "indicator"])
+    tb = tb.rename(columns={"value": "remittance_flows"})
+    tb["remittance_flows"] = tb["remittance_flows"].astype(float) * 1e6  # in million USD
+
+    # filter on 2021 as that is the year we have data for
+    tb = tb[tb["year"] == 2021]
+
+    tb.metadata.short_name = "bilateral_remittance"
+
+    #
+    # Harmonize country names
+    #
+    tb = geo.harmonize_countries(df=tb, country_col="country_origin", countries_file=paths.country_mapping_path)
```

Comment:
You could set warn_on_unused_countries to False for both of these

Reply:
Ah, that's a good idea, thank you!